src/pages/Atlas/index.tsx

```
@@ -0,0 +1,157 @@
+        cell: (ctx) => <PingButton domain={ctx.row.getValue('domain')} />,
+      })
+    ],
+    []
+  );
+
+  useEffect(() => {
+    const tableData: Array<RowData> = Object.values(gateways ?? {}).reduce(
+      (acc: Array<RowData>, gateway) => {
+        const totalStake = gateway.operatorStake + gateway.totalDelegatedStake
+        return [
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)